src/sbvr-api/sbvr-utils.coffee

```diff
@@ -826,16 +826,18 @@ exports.handleODataRequest = handleODataRequest = (req, res, next) ->
 # Reject the error to use the nice catch syntax
 constructError = (e) ->
 	Promise.reject(e)
-	.catch SbvrValidationError, BadRequestError, (err) ->
+	.catch SbvrValidationError, (err) ->
 		{ status: 400, body: err.message }
 	.catch PermissionError, (err) ->
 		{ status: 401, body: err.message }
 	.catch SqlCompilationError, TranslationError, ParsingError, PermissionParsingError, InternalRequestError, (err) ->
 		{ status: 500 }
 	.catch UnsupportedMethodError, (err) ->
 		{ status: 405, body: err.message }
-	.catch ForbiddenError, (err) ->
-		{ status: 403, body: err.message }
+	.catch HttpError, (err) ->
+		if err.status >= 500
```

Comment:
Is there any reason to drop the message for 500+? I feel like if a message has been attached then it's intentional? That said it's probably a sign that we should have a way to differentiate between internal error messages and ones we want to expose externally - and would also solve https://github.com/resin-io/pinejs/pull/168

Reply:
Not strong opinon on this, I just saw that we already are not attaching the message for 500 and thought that it was on purpose (I guessed that it was to avoid exposing internal details). Will change this.

Reply:
Specifically for the 500s where we ignore it's because they're unexpected and so they'll have an error message that could be dangerous to expose.  For explicitly thrown 500s then I think it's fine as they're expected and should only contain error messages that we're ok with showing to the user